### ISSUE TITLE
[x/gamm][StableSwap] Implement Scaling Factors to `CreateStableswapPool`

### ISSUE THREAD

**Issue by mattverse** [osmosis-labs/osmosis#1392]
With https://github.com/osmosis-labs/osmosis/pull/1354 being added, we now calculate different assets upon their precision points. 
We should be implementing scaling factors to `CreateStableswapPool`, `NewStableswapPool`, along with `ValidateBasic` if necessary.

**Comment by p0mvn** [comment]
This is blocked on deciding the future of: https://github.com/osmosis-labs/osmosis/pull/1459

**Comment by AlpinYukseloglu** [comment]
Seems like this is a duplicate of #2166 (please reopen if not)
